thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/wrapper/AnomalyDetectorWrapper.java

```diff
@@ -187,7 +187,7 @@ public DetectionPipelineResult run() throws Exception {
         LOG.warn("[DetectionConfigID{}] detecting anomalies for window {} to {} failed.", this.config.getId(), window.getStart(), window.getEnd(), e);
         lastException = e;
       }
-      anomalies.addAll(anomaliesForOneWindow);
+      anomalies.addAll(detectionResult.getAnomalies());
```

Comment:
I am confused here only the anomalies are merged.
What about the baselines in detectionResult?

Reply:
Those will be handled in a follow-up PR.